libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -145,6 +145,8 @@ class AP_Airspeed
         TYPE_I2C_MS4525=1,
         TYPE_ANALOG=2,
         TYPE_I2C_MS5525=3,
+        TYPE_I2C_MS5525_ADDRESS_0=4,
+        TYPE_I2C_MS5525_ADDRESS_1=5,
```

Comment:
Was this 0/1 already? If it was, I missed it, sorry.

Reply:
Fixed and pushed.